base/operators.jl

```diff
@@ -127,9 +127,78 @@ const .≤ = .<=
 const .≠ = .!=
 
 # core << >> and >>> takes Int as second arg
+"""
+    <<(x, n)
+
+Left bit shift operator, `x << n`. The result is `x` shifted left by `n` bits,
+where `n >= 0`, filling with `0`s. This is equivalent to `x * 2^n`.
+
+```jldoctest
+julia> Int8(3) << 2
+12
+
+julia> bits(Int8(3))
+"00000011"
+
+julia> bits(Int8(12))
+"00001100"
+```
+"""
 <<(x,y::Int)  = no_op_err("<<", typeof(x))
+
+"""
+    >>(x, n)
+
+Right bit shift operator, `x >> n`. The result is `x` shifted right by `n` bits, where
+`n >= 0`, filling with `0`s if `x >= 0`, `1`s if `x < 0`, preserving the sign of `x`.
+This is equivalent to `fld(x, 2^n)`.
+
+
+```jldoctest
+julia> Int8(13) >> 2
+3
+
+julia> bits(Int8(13))
+"00001101"
+
+julia> bits(Int8(3))
+"00000011"
+
+julia> Int8(-14) >> 2
+-4
+
+julia> bits(Int8(-14))
+"11110010"
+
+julia> bits(Int8(-4))
+"11111100"
+```
+"""
 >>(x,y::Int)  = no_op_err(">>", typeof(x))
+
+"""
+    >>>(x, n)
+
+Unsigned right bit shift operator, `x >>> n`. The result is `x` shifted right by `n` bits,
+where `n >= 0`, filling with `0`s.
+
+`BigInt`s are treated as if having infinite size, so no filling is required and this
```

Comment:
Minor: I wonder if the example should go before the bigint note?

This is a big improvement, looks great!